docs/publish/government-cloud-guidance.md

```diff
@@ -0,0 +1,39 @@
+---
+
+# Guidance for deploying on government clouds
+
+Microsoft provides the government cloud options for  our privacy sensitive customers in local, state, and national government organizations. This gives partners opportunities to target critical customers with their Office Add-ins. Due to the more restricted nature of these environments, which is important for the customers’ privacy and security needs, not all resources that are typically available in a standard production environment are available within these clouds.
+
+For partners providing their Office Add-ins to customers in these restricted cloud environments, there are important differences from the standard public cloud environment that must be considered. The following information gives the details that require special handling by developers writing Office Add-ins that target customers in these environments.
+
+## All sovereign environments
+
+For all government cloud (i.e. Sovereign Cloud) environments, the public Office Store is not available. This means that end-users are not able to acquire Office Add-ins directly from the public store. Administrators are also unable to deploy Office Add-ins directly from the public store into their Admin Portal. Instead, you must work with administrators to ensure the following:
+
+- The required resources and services for your solution are available inside the cloud boundary. This means that you have either worked with the tenant administrators to provision your service and resources inside of the cloud boundary, or you have worked with the network administrator to enable access to the your resources that reside outside of the cloud boundary.
+
+- The resources being accessed through the Office Add-in conform to the requirements of the government cloud from which they are being accessed. They also must conform to any additional requirements from the customer tenant for which the solution is being provisioned with regards to the transfer, management, and storage of potentially sensitive data, as well as code and resource security and access vetting procedures.
+
+- The Office Add-in manifest that describes the solution and its source location as applicable to the particular government cloud deployment is obtained from the partner and ingested for deployment to the appropriate users via the Admin Portal.
+
+## US Government Community Cloud (GCC)
+
+In addition to the above requirements applicable to all Sovereign Clouds, each Sovereign Cloud environment has its own considerations that may affect Office Add-ins targeting the environment. GCC is the least restrictive of the government cloud environments and some of the resources required by solutions built on top of Office are available from their usual production endpoints in this environment. One such resource is the office.js library. Solution partners should be able to continue to reference the public office.js resource as they do with their public production solution.
```

Comment:
```suggestion
In addition to requirements applicable to all Sovereign Clouds, each Sovereign Cloud environment has its own considerations that may affect Office Add-ins targeting the environment. GCC is the least restrictive of the government cloud environments and some of the resources required by the add-in are available from their usual production endpoints in this environment. One such resource is the Office JavaScript API library. Solution partners can continue to reference the public Office.js resource as they do with their public production solution.
```

Reply:
I want to be careful about how we refer to office.js here, since it's a specific resource. @jeremy-msft, what's the right phrasing here?

Reply:
I verified in the docs that we're using Office.js capitalized (except in code) and so I went with that. Here are two examples

![image](https://user-images.githubusercontent.com/30682803/196298102-50fd4b11-8a74-4e9c-9c1d-c084caac1308.png)

Reply:
Cool, thanks @lindalu-MSFT. I'll make those changes.